mslice/plotting/globalfiguremanager.py

```diff
@@ -226,8 +238,15 @@ def get_category(cls, num):
         return figure_category
 
     @classmethod
-    def set_figure_as_kept(cls, num):
+    def set_figure_as_kept(cls, num=None):
```

Comment:
Could you help me understand why `num` can now be `None`?

Reply:
I did this to simplify the code for the `KeepFigure` and `MakeCurrent` commands in #404.
When `figure_num` is `None` (This is only done using the `KeepFigure` and `MakeCurrent` commands), I set num to be the figure number of the last plot that was made and update the _last active_figure variable with num. This always denotes the number of the last figure plotted using mc.PlotCut(...) style commands etc. 
There is a need for a "_last_active_figure" for the cli as the globalfiguremanager sets the active figure to None when you mark a figure as kept, but we want to keep the reference to the last plot figure number as requested in #396.